### ISSUE TITLE
Is this project the GO version of facebook_page_analyzer?

### ISSUE THREAD

**Issue by SpecterHi** [0x46616c6b/suchbar#3]
Or it's a much powerful alternative of facebook_page_analyzer?

P.S.
Thanks for your great job and patience.
I asked you some questions about the facebook_page_analyzer, but I totally didn't know anything about Elastic at that time, LOL. Now I can understand how to use your tools.
I'm so glad to see this new project!

**Comment by 0x46616c6b** [comment]
Correct, I started to rewrite the code with golang to speedup fetching the data. 

Sure, to use that tool you need to know something about Elastic and how to query the data from this database. In the last week I added a helper for Grafana to add Dashboards for the pages.

In the future there are a lot of ideas in my head to visualize the data in a more complex way, so I will build a frontend for that purpose.

**Comment by SpecterHi** [comment]
Thanks! You did help a great lot! Looking forward to your up-coming masterpiece.
